### ISSUE TITLE
Ruby 2.7 deprecations

### ISSUE THREAD

**Issue by connorshea** [SeleniumHQ/selenium#7870]
## 🐛 Bug Report

There are deprecation warnings for Selenium in Ruby 2.7, which comes out next week. They're all from [this change](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/) to position arguments/kwargs.

The deprecation warning includes the callsite and the method being called:

```
/Users/connorshea/.rbenv/versions/2.7.0-preview3/lib/ruby/gems/2.7.0/gems/selenium-webdriver-3.142.6/lib/selenium/webdriver/chrome/driver.rb:43: warning: The last argument is used as the keyword parameter
/Users/connorshea/.rbenv/versions/2.7.0-preview3/lib/ruby/gems/2.7.0/gems/selenium-webdriver-3.142.6/lib/selenium/webdriver/remote/bridge.rb:44: warning: for `handshake' defined here

/Users/connorshea/.rbenv/versions/2.7.0-preview3/lib/ruby/gems/2.7.0/gems/selenium-webdriver-3.142.6/lib/selenium/webdriver/remote/bridge.rb:62: warning: The last argument is used as the keyword parameter
/Users/connorshea/.rbenv/versions/2.7.0-preview3/lib/ruby/gems/2.7.0/gems/selenium-webdriver-3.142.6/lib/selenium/webdriver/remote/w3c/bridge.rb:35: warning: for `initialize' defined here
```

These are the only two I'm seeing in my test suite, but there are probably others.

## To Reproduce

Detailed steps to reproduce the behavior:

It should be possible to reproduce this by just running the selenium-rb test suite on Ruby 2.7. I'm not sure exactly how to trigger these, short of running my app's test suite, unfortunately.

## Expected behavior

I'd expect no deprecation warnings.

## Environment

OS: macOS 10.14.6 Mojave
Browser: Chrome
Browser version: Chrome 79
Browser Driver version: ChromeDriver 79
Language Bindings version: Ruby 3.142.6

**Comment by p0deje** [comment]
Thanks for the report and the fix! I wonder if RuboCop has a cop that can identify all such cases.
Anyway, I've just released 3.142.7 with your fix.

**Comment by connorshea** [comment]
Thanks!
